src/tools/plan-generator.ts

```diff
@@ -117,6 +117,7 @@ export class PlanGenerator {
     const annotations = PlanGenerator.createAnnotations(connection.handle.annotations);
     const args = [handle, mode, type, annotations];
     if (connection.spec.expression) {
+      // TODO: Add a test for an expression once recipe2plan tests move to .cgtest
```

Comment:
Is there a bug (or GH issue) for this?

Reply:
No, but Shane is just doing it now as we speak.

Reply:
That works for me.